mods/default/chests.lua

```diff
@@ -265,29 +266,31 @@ function default.chest.register_chest(name, d)
 	def_closed.tiles[5] = def.tiles[3] -- drawtype to make them match the mesh
 	def_closed.tiles[3] = def.tiles[3].."^[transformFX"
 
-	minetest.register_node("default:" .. name, def_closed)
-	minetest.register_node("default:" .. name .. "_open", def_opened)
+	minetest.register_node(name, def_closed)
+	minetest.register_node(name .. "_open", def_opened)
 
 	-- convert old chests to this new variant
-	minetest.register_lbm({
-		label = "update chests to opening chests",
-		name = "default:upgrade_" .. name .. "_v2",
-		nodenames = {"default:" .. name},
-		action = function(pos, node)
-			local meta = minetest.get_meta(pos)
-			meta:set_string("formspec", nil)
-			local inv = meta:get_inventory()
-			local list = inv:get_list("default:chest")
-			if list then
-				inv:set_size("main", 8*4)
-				inv:set_list("main", list)
-				inv:set_list("default:chest", nil)
+	if modname == "default" then
+		minetest.register_lbm({
+			label = "update chests to opening chests",
+			name = modname .. ":upgrade_" .. itemname .. "_v2",
+			nodenames = {name},
+			action = function(pos, node)
+				local meta = minetest.get_meta(pos)
+				meta:set_string("formspec", nil)
+				local inv = meta:get_inventory()
+				local list = inv:get_list(modname .. ":chest")
+				if list then
+					inv:set_size("main", 8*4)
+					inv:set_list("main", list)
+					inv:set_list(modname .. ":chest", nil)
```

Comment:
this is always "default:chest"

Reply:
I figured that even when its always the same its better to use the variable than a hardcoded value, but its fine either way. I dropped the `modname` variable now.